src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -9,9 +9,7 @@
 
```

Comment:
why are the distance sensors removed entierely rather than removed when robot is a simulation